python/tests/test_ops.py

```diff
@@ -1402,6 +1402,27 @@ def test_stack(self):
         np_c = np.stack([np_a, np_b], axis=1)
         self.assertTrue(np.array_equal(c, np_c))
 
+    def test_linspace(self):
+        # Test default num = 50
+        a = mx.linspace(0, 1)
```

Comment:
This test case is failing, getting a shape of size 51 and there is an extra step past one at the end 
`
array([0, 0.0204082, 0.0408163, ..., 0.979592, 1, 1.02041], dtype=float32)
`